Sasquatch/SasquatchObjC/AppCenterDelegateObjC.m

```diff
@@ -71,6 +71,22 @@ - (NSString *)appSecretB2C {
 #endif
 }
 
+- (NSString *)appSecretFirebase {
+#if GCC_PREPROCESSOR_MACRO_PUPPET
+  return kMSPuppetFirebaseAppSecret;
+#else
+  return @"firebase blah";
```

Comment:
The naming for the macro and the other constants is kind of unclear so I'm not sure what these are supposed to be.